core/modules/wiki.js

```diff
@@ -179,7 +179,7 @@ exports.generateNewTitle = function(baseTitle,options) {
 		title = baseTitle;
 	while(this.tiddlerExists(title) || this.isShadowTiddler(title) || this.findDraft(title)) {
 		title = baseTitle + 
-			(options.prefix || " ") + 
+			(options.prefix === undefined ? " " : options.prefix) + 
```

Comment:
Hello @sukima,
It's just a matter of habit. If using null is better, then I can change it.

Hello @Jermolene , if we pass a number, is it converted into a string? or we should manage the conversion? if we have to convert it to a string maybe it is too complicated.